MerlinAU.sh

```diff
@@ -433,11 +433,11 @@ readonly CYANct="\e[1;36m"
 readonly WHITEct="\e[1;37m"
 
 ##----------------------------------------##
-## Modified by Martinski W. [2024-Feb-12] ##
+## Modified by Martinski W. [2024-Feb-21] ##
 ##----------------------------------------##
 readonly FW_Update_CRON_DefaultSchedule="0 0 * * 0"
 readonly CRON_DAYofWEEK_NAMES="(Sun|Mon|Tue|Wed|Thu|Fri|Sat)"
-readonly CRON_DAYofWEEK_RegEx="$CRON_DAYofWEEK_NAMES([,-]$CRON_DAYofWEEK_NAMES)*|[0-6]([,-][0-6])*"
+readonly CRON_DAYofWEEK_RegEx="$CRON_DAYofWEEK_NAMES([\/,-]$CRON_DAYofWEEK_NAMES)*|[0-6*]([\/,-][0-6])*"
```

Comment:
Is this the same friend that's been suggesting things since the start? Guy is good. Make sure you pass on the message!

Reply:
> Is this the same friend that's been suggesting things since the start? Guy is good. Make sure you pass on the message!
>
Yep, same guy. And yes, he's very good. I already took him out to lunch before for finding those other bugs & for making good suggestions for improvements. He's so grateful that we provided this script for F/W auto-update that he took me out to lunch a few days later.

Reply:
> > Is this the same friend that's been suggesting things since the start? Guy is good. Make sure you pass on the message!
> 
> Yep, same guy. And yes, he's very good. I already took him out to lunch before for finding those other bugs & for making good suggestions for improvements. He's so grateful that we provided this script for F/W auto-update that he took me out to lunch a few days later.

Just know id you were local at all you would be on the top of my priority list to take out for lunch on me hahaha!

Reply:
> 
> Just know if you were local at all you would be on the top of my priority list to take out for lunch on me hahaha!
>
I would do the same, especially during Christmastime when I would have taken you out for a couple of drinks as well :>).